factories/house.py

```diff
@@ -0,0 +1,19 @@
+import random
+
+import factory
+
+from house.models import House, Country, City, Job
+
+
+class HouseFactory(factory.Factory):
+    class Meta:
+        model = House
+
+    name = factory.Faker('name')
+    public = factory.Faker('boolean')
+    country = Country.objects.order_by('?')[0]
+    city = City.objects.order_by('?')[0]
+    parent_profession_1 = random.choice([c[0] for c in Job.choices])
+    parent_profession_2 = random.choice([c[0] for c in Job.choices])
+    income = random.randint(5000, 25000)
+    children = random.choice([i for i in range(8)])
```

Comment:
```suggestion
    name = factory.Faker('name')
    public = factory.Faker('boolean')
    country = factory.Iterator(Country.objects.all())
    city = factory.LazyAttribute(lambda x: random.choice(City.objects.filter(country=x.country)))
    parent_profession_1 = factory.Iterator(Job.choices)
    parent_profession_2 = factory.Iterator(Job.choices)
    income = factory.Faker('random_number', digits=5)
    children = factory.Faker('random_number', digits=1)
```

I tried using the script and it created the same field for all elements because random is used instead of the native factory methods after some research and trial error this suggestion creates different houses with different fields 
I will use the same techniques and create an expense factory and script PR soon

Reply:
![image](https://user-images.githubusercontent.com/61323123/162589946-5fbe2bdd-df0b-469a-9c58-2e1d012a13a8.png)
Thank you,!!!
I will fix it